frontend/src/app/universalReporting/LabReportForm.tsx

```diff
@@ -230,18 +237,32 @@ const LabReportForm = () => {
   };
 
   const submitForm = async () => {
-    const submissionResponse = await submitLabReport({
-      variables: {
-        patient: { ...patient },
-        provider: { ...provider },
-        facility: { ...facility },
-        specimen: { ...specimen },
-        testDetailsList: [...testDetailList],
-      },
-    });
-    setSubmissionResponse(
-      submissionResponse.data?.submitLabReport ?? "Response was null"
-    );
+    // TODO: add form validation
```

Comment:
Question - is validation "simple" enough that we could get away with just HTML5 form validation? (Though I don't think it'll block submission by default so idk if that's a good path)